cli/tests/unit/os_test.ts

```diff
@@ -185,3 +185,19 @@ unitTest({ perms: { env: false } }, function releasePerm(): void {
   }
   assert(caughtError);
 });
+
+unitTest({ perms: { env: true } }, function memoryInfo(): void {
+  assertNotEquals(Deno.memoryInfo(), {});
+});
+
+unitTest({ perms: { env: false } }, function memoryInfoPerm(): void {
+  let caughtError = false;
```

Comment:
Use assertThrows instead of ad-hoc exception capturing/asserting perhaps?

Reply:
Done 😉